mage_ai/orchestration/queue/process_queue.py

```diff
@@ -89,7 +89,10 @@ def __init__(self, queue: mp.Queue, job_dict):
                 self.dsn,
                 traces_sample_rate=SENTRY_TRACES_SAMPLE_RATE,
             )
+        if ENABLE_NEW_RELIC:
+            newrelic.agent.initialize('/home/src/newrelic.ini')
```

Comment:
same. let's not hardcode config path

Reply:
what happens if the newrelic config doesn't exist or is invalid, does the app still run correctly in that case?

Reply:
It fails if connfiguration missing. I now wrap it with a try catch clause to capture the configuration exception to make the code proceed